projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -66,6 +74,63 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
 
   private controlKeys: (keyof FormInterface)[] = [];
 
+  // instead of having the validators defined in the utils file
+  // we define them here to have access to the form types
```

Comment:
I think that really just means we should extract the types to their own file.

Reply:
The point was to get access to the form types without having to specify them when calling the validator but I hear your point... (CF next answer)